SmartEVSE2.X/EVSE.c

```diff
@@ -130,7 +130,7 @@ unsigned int CalcCurrent();
 
 
 // Configuration settings
-#pragma	config FCMEN = OFF,	IESO = OFF, PRICLKEN = ON
+#pragma	config FCMEN = ON, IESO = ON, PRICLKEN = ON                             // Enable Internal/External Oscillator Switchover
```

Comment:
Configuration is not (safely) writable on a firmware update. So no changes are possible.

Reply:
OK, yes they can be flashed on new devices this way. 
I'll test your changes when doing the INTOSC fallback test..